v2/internal/project/project.go

```diff
@@ -38,7 +38,7 @@ type Project struct {
 	/*** Internal Data ***/
 
 	// The path to the project directory
-	Path string
+	Path string `json:"projectdir"`
```

Comment:
The modification here is so that the test case can run normally, but I don't know if it will affect other code operations.